File: ts-sdk/src/sdk/models/operations/deleteanoauthtoken.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class DeleteAnOauthTokenPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=application_id" })
  applicationId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=token_id" })
  tokenId: string;
}

export class DeleteAnOauthTokenRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: DeleteAnOauthTokenPathParams;
}

export class DeleteAnOauthTokenResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: go-sdk/pkg/models/operations/deleteanoauthtoken.go
package operations

type DeleteAnOauthTokenPathParams struct {
	ApplicationID string `pathParam:"style=simple,explode=false,name=application_id"`
	Organization  string `pathParam:"style=simple,explode=false,name=organization"`
	TokenID       string `pathParam:"style=simple,explode=false,name=token_id"`
}

type DeleteAnOauthTokenRequest struct {
	PathParams DeleteAnOauthTokenPathParams
}

type DeleteAnOauthTokenResponse struct {
	ContentType string
	StatusCode  int
}
